javascript.js

```diff
@@ -1,18 +1,25 @@
 function emailCount() {
+  let uniqueEmails;
   let dataset = require('./email_log.json');
-
   let emails = dataset.emails;
 
-  let uniqueEmails = emails.reduce(function (previous, current) {
-    if (!previous.hasOwnProperty(current.email)) {
-      previous[current.email] = 1;
-    } else {
-      previous[current.email] += 1;
-    }
-    return previous;
-  }, {});
+  const getEmails = (function(){ 
```

Comment:
I'm not sure why you added the IIFE? You could get by with just using the `uniqueEmails` variable and setting that as the value on line 19.